content/docs/user-guide/project-structure/dvc-files.md

```diff
@@ -69,18 +69,18 @@ The following subfields may be present under `outs` entries:
-| `md5`<br/>`etag`<br/>`checksum` | Hash value for the file or directory being tracked with DVC. MD5 is used for most locations (local file system and SSH); [ETag](https://en.wikipedia.org/wiki/HTTP_ETag#Strong_and_weak_validation) for HTTP, S3, or Azure <abbr>external dependencies</abbr>; and a special _checksum_ for HDFS and WebHDFS. See `dvc import-url` for more information. |
-| `size`                          | Size of the file or directory (sum of all files).                                                                                                                                                                                                                                                                                                        |
-| `nfiles`                        | If this dependency is a directory, the number of files inside (recursive).                                                                                                                                                                                                                                                                               |
-| `repo`                          | This entry is only for external dependencies created with `dvc import`, and can contain `url`, `rev`, and `rev_lock` (detailed below).                                                                                                                                                                                                                   |
+| Field                           | Description                                                                                                                                                                                                                                                                                               |
+| ------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `path`                          | (Required) Path to the dependency (relative to `wdir`, which defaults to the file's location)                                                                                                                                                                                                             |
+| `md5`<br/>`etag`<br/>`checksum` | Only in <abbr>external dependencies</abbr> created with `dvc import-url`: Hash value of the imported file or directory. MD5 is used for most locations; [ETag](https://en.wikipedia.org/wiki/HTTP_ETag#Strong_and_weak_validation) for HTTP, S3, or Azure; and a special _checksum_ for HDFS and WebHDFS. |
```

Comment:
> MD5 is used for most locations

Does that mean specifically local, SSH and GCS? Cc @efiop thanks (just double checking) 🙏

Reply:
gcs uses etag too.

Reply:
OK, updated.